client/v2/autocli/msg.go

```diff
@@ -113,6 +116,26 @@ func (b *Builder) BuildMsgMethodCommand(descriptor protoreflect.MethodDescriptor
 			return err
 		}
 
+		clientCtx, err := config.ReadFromClientConfig(*b.ClientCtx)
```

Comment:
Not needed, we pass a clientCtx enriched with the config already (see root_v2.go)